acceptance/suites/pre_suite/puppet3_compat/30_install_dev_repos.rb

```diff
@@ -0,0 +1,21 @@
+case test_config[:puppetserver_install_type]
+when :package
+  step "Setup Puppet Server dev repository on the master." do
+    package_build_version = ENV['PACKAGE_BUILD_VERSION']
+    if package_build_version
+      install_puppetlabs_dev_repo master, 'puppetserver', package_build_version
+    else
+      abort("Environment variable PACKAGE_BUILD_VERSION required for package installs!")
+    end
+  end
+
+  puppet_build_version = test_config[:puppet_build_version]
```

Comment:
I would recommend at some point settling on this style of environment variable use (https://github.com/puppetlabs/puppet-server/blob/master/acceptance/lib/helper.rb#L30) or just using ENV['varname'] wherever necessary. I don't have any opinion about which is better, except to say that if you are using the variable value in more than one place it is probably beneficial to extract from the environment in a central location like the helper file.